app/views/membership.html

```diff
@@ -0,0 +1,292 @@
+<project-header class="top-header"></project-header>
+<project-page class="membership" ng-if="project">
+  <div class="middle-section">
+    <div class="middle-container">
+      <div class="middle-header">
+        <div class="container-fluid">
+          <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+        </div>
+      </div>
+      <div class="middle-content" persist-tab-state>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-12">
+              <h1>
+                Membership
+                <small class="pull-right meta">
+                  <a href="{{'roles' | helpLink}}">
+                    Learn more <i class="fa fa-external-link"></i>
+                  </a>
+                </small>
+              </h1>
+              <alerts alerts="alerts"></alerts>
+            </div>
+          </div>
+          <div  ng-if="!('rolebindings' | canI : 'list')">
+            <p>Nothing to see here, move along folks.</p>
```

Comment:
Since it is possible for someone to navigate to this page directly even after we deal with the navigation issues, we need a real error message here.  All it takes is someone sending a link to the membership page.